lib/Table/TableColumn.ts

```diff
@@ -97,6 +100,91 @@ export default class TableColumn {
     return result;
   }
 
+  /**
+   * Gets `math-expression-evaluator` tokens. These allow other column values to be used in an expression.
+   * Each token is generated by `transformation.dependencies` trait - which is an array of strings of column names.
+   * A token `x` is also added - which corresponds to the value in **this column**.
+   * For example:
+   * - if `dependencies = ['columnA']`
+   * - then a token will be generated for `columnA`
+   * - and then we can access the value in an expression (eg `x*columnA` will multiple each value in this column, by the value in `columnA`)
+   */
+  @computed
+  get mexpColumnTokens() {
+    return [
+      {
+        type: 3,
+        token: "x",
```

Comment:
I'm not sure, I think it makes it less readable adding a variable for `"x"`

Reply:
I am very much in the "all constants should be put in variables" school of thought, because 
- it makes them easier to rename
- easier to find - IDEs understand scoping of variables, and so give more useful results when you ask them to find all references to a variable rather than find all usages of a particular, fairly generic string.
- less prone to typos because the compiler can pick up the error.
- plus you can mouse over a variable to see its declaration anyway

but this one is very much a matter of opinion and it's not a hill I'm gonna die on.

Reply:
I have added `THIS_COLUMN_EXPRESSION_TOKEN`, but it has made documentation (especially in `lib\Traits\TableColumnTraits.ts`) quite hard to read.